build.rs

```diff
@@ -286,22 +351,19 @@ const ALLOWLIST_FN: &'static [&'static str] = &[
     "GLSLangGetNumUnpackedVaryingVectors",
 ];
 
+/// Change to one of the directory that contains moz.build
 fn fixup_path(path: &str) -> String {
```

Comment:
Hrm. This comment is a bit misleading because it suggests we are changing the "current working directory."

```suggestion
/// Make a path relative to the working directory that is used for the build.
fn fixup_path(path: &str) -> String {
```